docs/microservices.json

```diff
@@ -307,25 +597,469 @@
+            "spec": "https://hmcts.github.io/reform-api-docs/specs/ft-toggle-web.json",
+            "dependencies": [
+
+            ],
+            "apis": [],
+            "version": null
+        },
+        {
+            "id": "pbi-service",
+            "name": "RPE Private Beta Inivation",
+            "group": "RPE",
+            "proType": "api",
+            "description": "",
+            "repository": "https://github.com/hmcts/private-beta-invitation-service",
+            "spec": "https://hmcts.github.io/reform-api-docs/specs/private-beta-invitation-service.json",
+            "dependencies": [
+
+            ],
+            "apis": [],
+            "version": null
+        },
+        {
+            "id": "rpe-service-auth-provider",
+            "name": "RPE Serivce Auth Provider (S2S)",
+            "group": "RPE",
+            "proType": "api",
+            "description": "",
+            "repository": "https://github.com/hmcts/service-auth-provider-app",
+            "spec": "https://hmcts.github.io/reform-api-docs/specs/service-auth-provider-app.json",
+            "dependencies": [
+
+            ],
+            "apis": [],
+            "version": null
+        },
+        {
+            "id": "rpe-feature-toggle-api",
+            "name": "RPE Feature Toggle",
+            "group": "RPE",
+            "proType": "api",
+            "description": "",
+            "repository": "https://github.com/hmcts/feature-toggle-api",
+            "spec": "https://hmcts.github.io/reform-api-docs/specs/feature-toggle-api.json",
+            "dependencies": [
+                {
+                    "id": "rpe-service-auth-provider",
+                    "hard": true,
+                    "apis": []
+                }
+            ],
+            "apis": [],
+            "version": null
+        },
+        {
+            "id": "cmc-pdf-service",
+            "name": "CMC PDF Service",
+            "group": "RPE",
+            "proType": "api",
+            "description": "",
+            "repository": "https://github.com/hmcts/cmc-pdf-service",
+            "spec": "https://hmcts.github.io/reform-api-docs/specs/cmc-pdf-service.json",
+            "dependencies": [
+                {
+                    "id": "rpe-service-auth-provider",
+                    "hard": true,
+                    "apis": []
+                }
+            ],
+            "apis": [],
+            "version": null
+        },
+        {
+            "id": "draft-store-service",
+            "name": "RPE Draft Store",
+            "group": "RPE",
+            "proType": "api",
+            "description": "",
+            "repository": "https://github.com/hmcts/draft-store",
+            "spec": "https://hmcts.github.io/reform-api-docs/specs/draft-store.json",
+            "dependencies": [
+                {
+                    "id": "rpe-service-auth-provider",
+                    "hard": true,
+                    "apis": []
+                }
```

Comment:
I think draft store also depends on idam as it checks user details by making a call to it

Reply:
added